index.js

```diff
@@ -1,15 +1,13 @@
-const { TouchBar } = require('electron');
+const { TouchBar } = require('electron').remote;
 const { TouchBarButton } = TouchBar;
 
-if (TouchBar) {
```

Comment:
In my opinion is not necessary. This plugin should only be used if you have a touch bar.